pkg/api/api.proto

```diff
@@ -234,7 +251,10 @@ enum Event {
   POST_UPDATE_CONTAINER = 9;
   STOP_CONTAINER = 10;
   REMOVE_CONTAINER = 11;
+  // TODO: Should this be moved to the end (assuming it's not used anywhere)?
```

Comment:
This is used for event mask validation and String()ification, so it should be updated to really be the last one.